gradle.properties

```diff
@@ -24,6 +24,6 @@ publishChannels=Stable
 # Uncomment one of the following settings: either pycharmPath or pythonPlugin
 #######################################################################################################################
 # Run Mypy plugin inside PyCharm installed into the following path
-pycharmPath=/home/leinardi/pycharm-community
+# pycharmPath=/home/leinardi/pycharm-community
```

Comment:
Please do not comment this: PyCharm should be the default target of this plugin.
If you commented it to make the test run on CI, it would be better to have some logic to not set it only if the build is running on CI...

Reply:
If I leave this in, gradle fails with the following error:

```
Execution failed for task ':compileJava'.
> Could not resolve all files for configuration ':compileClasspath'.
   > Could not find :pycharm:.
     Searched in the following locations:
       - file:/home/leinardi/pycharm-community/lib/pycharm.jar
     Required by:
         project :
```

It seems weird to assume that every developer of this plugin has username `leinardi` :smile: and having to manually download the IDE to a certain file path is inconvenient as well.

Are there any alternatives?

Reply:
Yeah, this was just me being lazy 😅. But I think that, if you want to develop a plugin for PyCharm, it is not so weird that it is required to have a local copy of PyCharm.
Probably a better way to do it would be to let the dev define the path to PyCharm inside the `~./gradle/gradle.properties` or to provide it via environmental variable...
But, as a quick hack, you can simply change this path inside the project's `gradle.properties` and not commit it.
But, please, remember to test the plugin changes with PyCharm: like I said I had issues in the past where the plugin worked fine on Idea but did not work on PyCharm...

Reply:
After some digging, it seems I can set `ideaVersion=PC-2021.2.3`, which automatically downloads PyCharm Community Edition.

This should resolve your concerns about PyCharm/IntelliJ compatibility? So I think `pycharmPath` should remain commented out in the default config.

Reply:
Oh interesting! So, when you run the plugin from the IDE, for example to debug, it opens PyCharm, not IDEA, right?

Reply:
Yep.